repository.rdf

```diff
@@ -417,16 +336,6 @@
 
 
 
-    <!-- http://fedora.info/definitions/v4/repository#Configuration -->
-
-    <owl:Class rdf:about="http://fedora.info/definitions/v4/repository#Configuration">
```

Comment:
@whikloj , you are the last maintainer standing for `-transform`. Any objections?